example/components/Login.vue

```diff
@@ -6,7 +6,6 @@
 			<h1>You are authenticated</h1>
 		</div>
 		<Descope
-			v-else
```

Comment:
why removed?

Reply:
but doesn't it means that the Descope will be rendered also if user is authenticated?

Reply:
It's on the login page, I think it's reasonable to render it
The alternative is that the success event will be dispatched before the session is updated 
which will lead us to the same problem as before